File: ChameraVoteWebClient/scripts/RSA.js
class RSA
{
    static n = 35;
    static PublicKey = [7,RSA.n];
    static ModInverse = function(a, b) {
        a %= b;
        for (var x = 1; x < b; x++) {
            if ((a*x)%b == 1) {
                return x;
            }
        }
    }
}

File: ChameraVote/Utility/RSA.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ChameraVote.Utility
{
    public class RSA
    {
        public static int n = 21;

        public static Tuple<int, int> publicKey = new Tuple<int, int>(23,RSA.n);

        public static int ModInverse(int a, int n)
        {
            int i = n, v = 0, d = 1;
            while (a > 0)
            {
                int t = i / a, x = a;
                a = i % x;
                i = x;
                x = d;
                d = v - t * x;
                v = x;
            }
            v %= n;
            if (v < 0) v = (v + n) % n;
            return v;
        }
    }
}
